h39s/AQUA
# JAT-102 Setup fallback for non-case-sensitive file systems

# Summary
Add functionality to adjust preset name case sensitivity to match the file system rules.

## Changes
- Add a global flag `isCaseSensitiveFs` to `IState` with a default of `false`
- Set flag to `true` if we are able to successfully enable case sensitivity in the preset folder
- Add case insensitive validation for the following cases:
   - Creating a new preset
   - Saving to an existing preset
   - Renaming an existing preset

## Miscellaneous
- Fix bug where users could attempt to save erroneous preset names by pressing `Enter`
- Improve styling of preset list in the case where an unbreakable word overflows horizontally

## Tests
- Add tests to validate preset name validation
- Add test to ensure that text inputs can't be submitted using the `Enter` button when it is in a disabled state
